assets/templates/layouts.md-password-recovery.hbs

```diff
@@ -0,0 +1,14 @@
+<div class=login-main>
+    <div class=login-window>
+        <h3 class=login-title>Forgot your password?</h3>
+        <div class="alert">Enter your email to request a password reset</div>
+        <div class=login-body>
+            <label><span>Email Address</span><input type=text placeholder="e.g. email@email.com"></input></label>
+        </div>
+        <div class=login-actions>
+
+            <button class="button-link">CANCEL</button>
```

Comment:
If these are supposed to be all caps it should probably be done using a css text-transform property